### PR TITLE
added NuGet option to splunk-otel-dotnet-docker example

### PR DESCRIPTION
**Pull request by dmitchsplunk**
Some customers have expressed an interest in using the NuGet package installation option to add instrumentation to their .NET applications running in Docker images.  The proposed changes add this alternative option to the existing splunk-otel-dotnet-docker example, which is already linked to from the Splunk documentation. 

I have read the CLA Document and I hereby sign the CLA

I have read the Code of Conduct and I hereby accept the Terms

### CODE REVIEW

**Comment by dmitchsplunk** [comment]
recheck

**Comment by dmitchsplunk** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by dmitchsplunk** [comment]
I have read the Code of Conduct and I hereby accept the Terms

**Review by greatestusername-splunk — APPROVED** [review]
This is awesome! Thank you for contributing this additional example and docs update

File: integration-examples/splunk-otel-dotnet-docker/MultiStageDockerNuGetOption/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+ARG BUILD_CONFIGURATION=Release
+WORKDIR /src
+COPY ["MultiStageDocker/MultiStageDocker.csproj", "MultiStageDocker/"]
+RUN dotnet restore "./MultiStageDocker/./MultiStageDocker.csproj"
+WORKDIR "/src/MultiStageDocker"
+COPY . .
+
+RUN dotnet add "./MultiStageDocker.csproj" package Splunk.OpenTelemetry.AutoInstrumentation --prerelease
```
Comment (RassK) [inline integration-examples/splunk-otel-dotnet-docker/MultiStageDockerNuGetOption/Dockerfile:16]: Could be a bit painful to developer experience.  More explained in Slack.

**Review by RassK — APPROVED** [review]

**Review by greatestusername-splunk — APPROVED** [review]
Thanks for this contribution!

**Pull request merged**